README.md

```diff
@@ -50,31 +50,22 @@ the central point of interaction for users:
   an update mechanism so there will only need to be a single store xdc app in
   the store bot chat. 
 
-## Setting Up the Bot
+## Setting up the bot
 
-### Requirements
-
-You need to have `rust` and `node` installed.
-Optionally, have [pnpm](https://pnpm.io/) installed.
-
-### Setting up the bot 
+### Installation
 
-Go to the `frontend` folder and install the packages using `pnpm install` or `pnpm i`.
-If you do not have `pnpm` installed, you can run `npx pnpm install` to use `pnpm` 
-without installing it globally or use `npm install` instead.
-However, if you use `npm install`, the latest versions of dependencies will be installed
-instead of the ones listed in the `frontend/pnpm-lock.yaml` file.
+TODO: Download the latest release from https://github.com/webxdc/store/releases,
+extract the compressed archive to the desired location, it containts the bot command line
+program and required assets.
```

Comment:
it's nicer to provide an example CLI invocation after download.

Reply:
the example invocation is in the next step "running the bot" this first step is about getting the binary only

Reply:
but we could for example, tell reader to run `./xdcstore --version` to check the program is "installed" correctly

Reply:
done: `/path/to/xdcstore version`